src/services/azblob/backend.rs

```diff
@@ -730,11 +812,172 @@ impl AzblobBackend {
 
         self.client.send_async(req).await
     }
+
+    async fn azblob_batch_del(
```

Comment:
Names are always the most important aspect. At OpenDAL, we prioritize clear and easily understandable names. Have you considered using `azblob_batch_delete`?